### DISCUSSION TITLE
The name of the NFT collection with an unnecessary postscript

### DISCUSSION THREAD

**Discussion by Amnamot** [phantom/docs#90] [Help & Support]
### Summary

Hello!

After the token is minted, it is displayed in the Phantom with the correct collection name. But after some time, the name of the collection is supplemented by the name of the Magic Eden marketplace.

Please tell me why is this happening?
Why, for example, not Opensea?
How to get rid of it?

### Steps to Reproduce

![image](https://github.com/phantom/docs/assets/108093351/eaa070b7-6e98-462b-b871-e35264d0d965)

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hello @Amnamot,

Thank you for writing in!

Can you provide the token account for the token that is displaying this behavior, please?
